src/Uno.Extensions.Reactive.UI/Presentation/Bindings/Bindable.cs

```diff
@@ -36,7 +56,7 @@ public class Bindable<T> : IBindable, INotifyPropertyChanged, IFeed<T>
 	/// Is so, the <see cref="PropertyChanged"/> will be raise accordingly.
 	/// </param>
 	/// <exception cref="ArgumentException">If the property is invalid</exception>
-	public Bindable(BindablePropertyInfo<T> property, bool hasValueProperty = false)
+	protected Bindable(BindablePropertyInfo<T> property, bool hasValueProperty)
```

Comment:
You are breaking the compatibilty, no?

@jeromelaban, I think we should add the public signatures validator in this repo...

Reply:
Bin compatibility yes, code compatibility not really: I moved an optional parameter which was designed to be used only by sub-class into a dedicated properted ctor.

Anyway this class is designed to be used by code-gen only (I noticed that I forgot the DebuggerBrowsable.Never on it).

But I do agree that we should have a public API validator on this project. (I would override it for this case anyway 😁)